pkgs/development/libraries/aribb25/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitLab, fetchpatch, autoreconfHook, pkgconfig, pcsclite
+, PCSC }:
+
+stdenv.mkDerivation rec {
+  pname = "aribb25";
+  version = "0.2.7";
+
+  src = fetchFromGitLab {
+    domain = "code.videolan.org";
+    owner = "videolan";
+    repo = pname;
+    # rev = version; FIXME: uncomment in next release
+    rev = "1kb9crfqib0npiyjk4zb63zqlzbhqm35nz8nafsvdjd71qbd2amp"; # 0.2.7 with build fixes
```

Comment:
Copy-paste error?
```suggestion
    rev = "c14938692b313b5ba953543fd94fd1cad0eeef18"; # 0.2.7 with build fixes
```

Reply:
You're right. Looks like the build was succeeding on my local machine because the sha256 value matched the source that was already in the Nix store.